Roomz/src/main/java/com/google/sps/servlets/CreateProfileServlet.java

```diff
@@ -0,0 +1,46 @@
+package com.google.sps.servlets;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.FullEntity;
+import com.google.cloud.datastore.KeyFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+
+/** Servlet responsible for creating new user profile. */
+@WebServlet("/new-profile")
+public class CreateProfileServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    // Sanitize user input to remove HTML tags and JavaScript.
+    String fullname = Jsoup.clean(request.getParameter("fullname"), Whitelist.none());
+    String email = Jsoup.clean(request.getParameter("email"), Whitelist.none());
+    String bio = Jsoup.clean(request.getParameter("bio"), Whitelist.none());
+    String age = Jsoup.clean(request.getParameter("age"), Whitelist.none());
+    String gender = Jsoup.clean(request.getParameter("gender"), Whitelist.none());
```

Comment:
this might be a good opportunity to refactor the repeated code into a helper function

Reply:
Hi Alan, I'm not exactly sure what a helper function is, do you mean I can add the repetitive code in a new function and pass the properties as parameters instead of writing this line every time?